src/EditModel/EditDataEntryForm/useResize.js

```diff
@@ -0,0 +1,37 @@
+import { useRef, useCallback } from 'react';
+import { clampPaletteWidth } from './clampPaletteWidth';
+
+export const useResize = ({ paletteWidth, setPaletteWidth }) => {
```

Comment:
This was extracted from https://github.com/dhis2/maintenance-app/blob/224f2ec40242776d1e92ab5d6fdc2cc9d4989c88/src/EditModel/EditDataEntryForm.component.js#L364-L369 and https://github.com/dhis2/maintenance-app/blob/224f2ec40242776d1e92ab5d6fdc2cc9d4989c88/src/EditModel/EditDataEntryForm.component.js#L211-L229